### PR TITLE
Feature/favorite exercise

### PR DESCRIPTION
**Pull request by esrefbozkurt**
add fav icons and toggle them

### CODE REVIEW

File: components/Main/Exercises/Favorite/Favorite.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+import { useState } from "react";
+import { StyledFav } from "./StyledFav";
+
+export default function Favorite() {
+  const [favourite, setFavourite] = useState(false);
```
Comment (onebarloop) [inline components/Main/Exercises/Favorite/Favorite.js:6]: I assume you will later have to uplift this state to a higher level, to be able to conditional render the favorites, depending on this state? But right now, you "only" toggle the buttons with it, which ist fine.

**Review by onebarloop — APPROVED** [review]
Nice and very readable! I like your approach to outsource the styled components. Maybe I will adapt this :)

File: components/Main/Exercises/Favorite/Favorite.js (hunk 1 above)
Comment (esrefbozkurt) [reply components/Main/Exercises/Favorite/Favorite.js:6]: Thäänks

Comment (esrefbozkurt) [reply components/Main/Exercises/Favorite/Favorite.js:6]: Yes will do this in the next US.

**Pull request merged**